data/data/appeals-service-api/006-listedBuilding.json

```diff
@@ -0,0 +1,17 @@
+[
+	{
+		"name": "10 and 10A Special House",
+		"reference": "1010101",
+		"listed-building-grade": "II"
+	},
+	{
+		"name": "AN IMPORTANT BUILDING",
+		"reference": "1010102",
+		"listed-building-grade": "II*"
```

Comment:
Do we need to camel case this or are we going to use this as a test data set to be processed by the function?

Reply:
yes sorry